listenbrainz/db/tests/test_feedback.py

```diff
@@ -58,32 +58,36 @@ def insert_test_data_with_metadata(self, user_id, neg_score=False):
         """ Insert test data with metadata into the database """
 
         with msb_db.engine.connect() as connection:
-            msid = submit_recording(connection, self.sample_recording)
+            msid = get_id_from_meta_hash(connection, self.sample_recording)
```

Comment:
Why is this needed especially why in test only?

Reply:
Running tests repeatedly (using test -u) causes this test to fail with a duplicate row violation. Checking to see if the data is already in the DB prevent this error.

Reply:
Oh, weird. because submit_recording is supposed to take care of that. Anyways fine for a test. We can investigate later.